global_helpers/panther_greynoise_helpers.py

```diff
@@ -42,19 +40,30 @@ def advanced_only():
 
         return advanced_only()
 
-    def ip_address(self, match_field) -> str:
-        return deep_get(self.noise, match_field, "ip")
+    def subscription_level(self):
+        return self.sublevel
+
+    def ip_address(self, match_field: str) -> list or str:
```

Comment:
throughout this PR you'll want to find replace `list or str` with `Union[list[str], str]`

Reply:
you'll also need to add
```
from typing import Union
```

Reply:
Fixed